benchmark/attri_util.py

```diff
@@ -180,14 +182,23 @@ def __str__(self) -> str:
                 f"{'Size Detail':>20}\n"
                 f"{'-' * 120}\n"
             )
+        elif self.result[0].gbps is not None:
+            header = (
+                f"\nOperator: {self.op_name}  Performance Test (dtype={self.dtype}, mode={self.mode}, "
+                f"level={self.level})\n"
+                f"{'Size':<10} {'Torch Latency (ms)':>20} {'Gems Latency (ms)':>20} {'Gems Speedup':>20} {'Torch GBPS ':>20} {'Gems GBPS':>20}"  # noqa
+                f"{'Size Detail':>20}\n"
+                f"{'-' * 150}\n"
+            )
```

Comment:
I think we may use `tabulate` to print tables inthe future.

Reply:
I'll leave this to someone else..